packages/@okta/i18n/src/properties/login.properties

```diff
@@ -959,3 +959,7 @@ oie.consent.enduser.deny.description = You may close this window at any time.
 
 # Browser thrown errors
 oie.browser.error.NotAllowedError = The operation either timed out or was not allowed.
+
+# soft_token API errors
+authfactor.challenge.soft_token.invalid_passcode = Your code doesn't match our records. Please try again.
```

Comment:
This followed backend pattern and v1 api error key pattern, we should not change the key , so cannot add `oie` in the beginning